src/app/store/index.js

```diff
@@ -183,6 +183,15 @@ const migrateStore = () => {
     if (semver.lt(cnc.version, '1.9.16')) {
         store.unset('widgets.axes.jog.step');
     }
+
+    // 1.9.28
```

Comment:
I might have messed up the last release 😬 apologies.

https://github.com/cncjs/cncjs/releases/tag/v1.9.28 already exists, but I failed to increment package.json accordingly. Some documentation for release procedure would help, but that's on me.

If you wish to target the next release, 1.9.29 might be better.

Reply:
The official version of CNCjs is determined by publishing a release on NPM. You can view the available versions on the following link: 

https://www.npmjs.com/package/cncjs?activeTab=versions
![image](https://user-images.githubusercontent.com/447801/213118329-43be3a4b-2b14-40d8-8f37-d8ee12270f4c.png)

As the recent changes do not include any new features and there is no commit for version bump, I propose using current version 1.9.28 for the next release to include PR #796 and PR #800.